src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+        <location filename="../../rebindwidget.ui" line="1671"/>
+        <source>Apply</source>
+        <translation>Anwenden</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="458"/>
+        <source>Key %1 (%2) is pressed but never released.
+This will result in the key being pressed by the macro until you manually press the key itself and release it.
+
+Are you sure you want to continue?</source>
+        <translation>Taste %1 (%2) wird gedrückt, aber nicht losgelassen.
+Dies führt dazu, dass die Taste vom Makro gedrückt wird, bis Sie die Taste selbst manuell drücken und loslassen.
+
+Sind Sie sicher, dass Sie fortfahren möchten?</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="461"/>
+        <source>Key %1 (%2) is released but never pressed.
+This will have no observable effect unless the key is held down manually or by another macro.
+
+Are you sure you want to continue?</source>
+        <translation>Die Taste %1 (%2) wurde losgelassen, aber nie gedrückt.
+Dies hat keine erkennbaren Auswirkungen, es sei denn, die Taste wird manuell oder durch ein anderes Makro gedrückt gehalten.
+
+Sind Sie sicher, dass Sie fortfahren möchten?</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="464"/>
+        <source>Macro warning</source>
+        <translation>Makro-Warnung</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="810"/>
+        <source>Click Apply or manually edit the events.</source>
+        <translation>Auf Anwenden klicken oder Ereignisse manuell bearbeiten.</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="848"/>
+        <source>Unknown key combination pressed</source>
+        <translation>Unbekannte Tastenkombination gedrückt</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="848"/>
+        <source>An unknown key combination (%1, %2) has been pressed.
+Make sure your keyboard layout is set to English - United States while recording macros.</source>
+        <translation>Es wurde eine unbekannte Tastenkombination (%1, %2) gedrückt.
+Vergewissern Sie sich, dass Ihre Tastaturbelegung auf Englisch - Vereinigte Staaten eingestellt ist, während Sie Makros aufzeichnen.</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="858"/>
+        <source>Stop Recording</source>
+        <translatorcomment>Alternative: stoppen</translatorcomment>
+        <translation>Aufzeichnung anhalten</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="859"/>
+        <source>Type your macro and press Stop Recording when finished.</source>
+        <translation>Makro eingeben und danach auf Aufzeichnung anhalten klicken.</translation>
```

Comment:
I would put the "Aufzeichnung anhalten" either in quotes or otherwise mark them up, because the english version highlights it by using it's title case (kind of).